contrib/nginx/template.py

```diff
@@ -0,0 +1,72 @@
+pkgname = "nginx"
+pkgver = "1.24.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--prefix=/etc/nginx",
+    "--sbin-path=/usr/bin/nginx",
+    "--modules-path=/usr/libexec/nginx",
+    "--conf-path=/etc/nginx/nginx.conf",
+    "--http-log-path=/var/log/nginx/access.log",
+    "--error-log-path=/var/log/nginx/error.log",
+    "--pid-path=/run/nginx.pid",
+    "--with-compat",
+    "--with-pcre",
+    "--user=_www",
+    "--group=_www",
```

Comment:
not the biggest unix group separator person myself but maybe --user should be _nginx? _www is probably intended to be shared with a bunch of stuff

Reply:
It was discussed with q66 on IRC and he suggested to add _www user first on nginx and then if we add another webserver, then we move its creation to another place

Reply:
that makes sense :)